# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/nsWindow.h
# Commit: 2a39a8e425b0
# Full Hash: 2a39a8e425b0824c670ba03452c437c3748c990b
# Author: Greg V <greg@unrelenting.technology>
# Date: 2020-01-06 21:54:03
# Regressor Bug: 1587360
# File Overlap Count: 1
# Description:
#   Bug 1587360 - Add support for idle-inhibit Wayland protocol r=stransky
#   
#   The idle-inhibit protocol allows idle wake locks on Wayland
#   without any of the supported D-Bus interfaces running.
#   
# ==============================================================================

diff -r 47592189b305 -r 2a39a8e425b0 widget/gtk/nsWindow.h
--- a/widget/gtk/nsWindow.h	Mon Jan 06 17:02:14 2020 +0000
+++ b/widget/gtk/nsWindow.h	Mon Jan 06 16:11:19 2020 +0000
@@ -230,6 +230,7 @@
 
 #ifdef MOZ_WAYLAND
   void SetEGLNativeWindowSize(const LayoutDeviceIntSize& aEGLWindowSize);
+  static nsWindow* GetFocusedWindow();
 #endif
 
   RefPtr<mozilla::gfx::VsyncSource> GetVsyncSource() override;